setup.cfg

```diff
@@ -20,19 +20,14 @@ python_requires = >=3.7
 install_requires =
     aiohttp>=3.6,<4
     protobuf>=3.0,<4
+    aiofiles>=0.6.0<1
 setup_requires =
     setuptools_scm
 
-[options.extras_require]
```

Comment:
I would keep this as is might be convenient later. I also though the build processes uses it in the CI pipeline.

Reply:
no it uses the requirments.txt. 
thats actually why i moved it there and removed this  section, because we ended up having dependencies declared at different places.